include/RAJA/policy/sequential/forall.hpp

```diff
@@ -92,6 +92,8 @@ RAJA_INLINE void forall(const seq_exec &, Iterable &&iter, Func &&body)
   auto begin = std::begin(iter);
   auto end = std::end(iter);
   auto distance = std::distance(begin, end);
+
+  //RAJA_NO_SIMD
```

Comment:
Why commented out?

Reply:
Sorry about that, this branch started failing travis build tests and I am trying to trace back the problem.